sites/kit.svelte.dev/src/routes/home/Showcase.svelte

```diff
@@ -4,7 +4,7 @@
 	import pudding from './showcase/pudding.png';
 	import pocketbase from './showcase/pocketbase.png';
 	import pronauns from './showcase/pronauns.png';
-	import dropzone from './showcase/dropzone.png';
```

Comment:
i'll leave it up to rich, but I might personally leave dropzone.png in the repo even if unused. we had talked about creating a kit.svelte.dev/showcase with a larger set of sites and we could include include dropzone there still